xbmc/video/VideoDatabase.cpp

```diff
@@ -2031,7 +2031,7 @@ int CVideoDatabase::SetDetailsForMovie(const std::string& strFilenameAndPath, co
   try
   {
     BeginTransaction();
-
+    
```

Comment:
That wasn't meant to slip in there, probably leftovers from trying to trace the problem I mentioned above